packages/opentelemetry-sdk-trace-base/src/export/WriteableSpan.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as api from '@opentelemetry/api';
+import { SpanAttributeValue } from '@opentelemetry/api';
+import {
+  InstrumentationLibrary
+} from '@opentelemetry/core';
+import { IResource } from '@opentelemetry/resources';
+import { TimedEvent } from '../TimedEvent';
+
+export interface WriteableSpan extends api.Span {
```

Comment:
I think this interface also extends `ReadableSpan`. Should we add it to the declaration?
If we `extends ReadableSpan`, then we don't even need to state the interface members in the body.